plugins/modules/firewalld.py

```diff
@@ -78,28 +78,28 @@
   zone:
     description:
       - The firewalld zone to add/remove to/from.
-      - Note that the default zone can be configured per system but C(public) is default from upstream.
+      - Note that the default zone can be configured per system but V(public) is default from upstream.
       - Available choices can be extended based on per-system configs, listed here are "out of the box" defaults.
-      - Possible values include C(block), C(dmz), C(drop), C(external), C(home), C(internal), C(public), C(trusted), C(work).
+      - Possible values include V(block), V(dmz), V(drop), V(external), V(home), V(internal), V(public), V(trusted), V(work).
     type: str
   permanent:
     description:
       - Whether to apply this change to the permanent firewalld configuration.
       - As of Ansible 2.3, permanent operations can operate on firewalld configs when it is not running (requires firewalld >= 0.3.9).
-      - Note that if this is C(false), I(immediate) defaults to C(true).
+      - Note that if this is V(false), O(immediate=true).
```

Comment:
From the old formulation, if the user provides an explicit value for `immediate`, it's not set to `true`:
```suggestion
      - Note that if this is V(false), O(immediate=true) by default.
```